modules/stats.php

```diff
@@ -1031,7 +1031,8 @@ function stats_dashboard_widget_content() {
 
 	$post_ids = array();
 
-	$csv_args = array( 'top' => '&limit=8', 'search' => '&limit=5' );
+	$csv_end_date = date( 'Y-m-d', current_time( 'timestamp' ) );
```

Comment:
Why `date( 'Y-m-d', current_time( 'timestamp' ) )` instead of `date( 'Y-m-d', time() )` or even just `date( 'Y-m-d' )`?